nixos/modules/misc/ids.nix

```diff
@@ -275,6 +275,8 @@
       gocd-server = 252;
       terraria = 253;
       mattermost = 254;
+      quagga = 255;
```

Comment:
I see. Using static ids ensures that data is not suddenly owned by a different user/group due to variability in ids across NixOS systems/activations. Whether this matters depends on the nature of the data (is it expected to be portable, are there privacy/security concerns, &c) and the amount of manual intervention involved to resolve inconsistencies.

Another reason to use static ids is if you need to refer to the user/group before the id/name mapping is established.

If you think it is appropriate for this service, by all means use a static id, but keep in mind that it is a finite resource.